core/templates/pages/exploration-editor-page/services/exploration-states.service.ts

```diff
@@ -661,6 +667,14 @@ export class ExplorationStatesService {
     this.saveStateProperty(stateName, 'linked_skill_id', newLinkedSkillId);
   }
 
+  saveInapplicableSkillMisconceptionIds(stateName: string, s: string[]): void {
```

Comment:
nit: please use a meaningful variable name -- it is not clear what "s" represents.